connectors/snowflake-sink/2.10.3/snowflake-sink.md

```diff
@@ -0,0 +1,385 @@
+
+**Example**
+
+* JSON
+
+   ```json
+    {
+        "tenant": "public",
+        "namespace": "default",
+        "name": "snowflake-sink",
+        "archive": "connectors/pulsar-io-snowflake-{{connector:version}}.nar",
+        "inputs": [
+          "test-snowflake-pulsar"
+        ],
+        "parallelism": 1,
+        "retainOrdering": true,
+        "processingGuarantees": "ATLEAST_ONCE",
+        "sourceSubscriptionName": "sf_sink_sub",
+        "configs": {
+          "user": "TEST",
+          "host": "ry77682.us-central1.gcp.snowflakecomputing.com:443",
+          "schema": "DEMO",
+          "warehouse": "SNDEV",
+          "database": "TESTDB",
+          "privateKey": "SECRETS"
+      }
+    }
+    ```
+
+* YAML
+
+    ```yaml
+    tenant: public
+    namespace: default
+    name: snowflake-sink
+    parallelism: 1
+    inputs:
+      - test-snowflake-pulsar
+    archive: connectors/pulsar-io-snowflake-{{connector:version}}.nar
+    sourceSubscriptionName: sf_sink_sub
+    retainOrdering: true
+    processingGuarantees: ATLEAST_ONCE
+    configs:
+      user: TEST
+      host: ry77682.us-central1.gcp.snowflakecomputing.com:443
+      schema: DEMO
+      warehouse: SNDEV
+      database: TESTDB
+      privateKey: SECRETS
+    ```
+
+## Work with Function Mesh
+You can create a [CustomResourceDefinitions (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to create a Snowflake sink connector. Using CRD makes Function Mesh naturally integrate with the Kubernetes ecosystem. For more information about Pulsar sink CRD configurations, see [sink CRD configurations](https://functionmesh.io/docs/connectors/io-crd-config/sink-crd-config).
+
+You can define a CRD file (YAML) to set the properties as below.
+
+```yaml
+apiVersion: compute.functionmesh.io/v1alpha1
+kind: Sink
+metadata:
+  name: snowflake-sink-sample
+spec:
+  image: streamnative/pulsar-io-snowflake:{{connector:version}}
+  replicas: 1
+  maxReplicas: 1
+  retainOrdering: true
+  input:
+    topics: 
+      - persistent://public/default/test-snowflake-pulsar
+  sinkConfig:
+    user: TEST
+    host: ry77682.us-central1.gcp.snowflakecomputing.com:443
+    schema: DEMO
+    warehouse: SNDEV
+    database: TESTDB
+    privateKey: SECRETS
+  pulsar:
+    pulsarConfig: "test-pulsar-sink-config"
+  resources:
+    limits:
+      cpu: "0.2"
+      memory: 1.1G
+    requests:
+      cpu: "0.1"
+      memory: 1G
+  java:
+    jar: connectors/pulsar-io-snowflake-{{connector:version}}.nar
+  clusterName: test-pulsar
+  autoAck: false
+```
+
+# How to use
+
+You can use the Snowflake sink connector with Function Worker or Function Mesh.
+
+## Work with Function Worker
+
+You can use the Snowflake sink connector as a non built-in connector or a built-in connector as below.
```

Comment:
"non built-in" sounds a bit weird to me. Can we use "standalone" or "freestanding" or something else instead?

Reply:
Replace it by `standalone `.